cmd/manager/main.go

```
@@ -74,6 +75,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := utils.EnsureLxcfsParentDir(*lxcfsPath); err != nil {
```

golangci-lint (1.64.8)

shadow: declaration of "err" shadows declaration at line 72

(govet)